### PR TITLE
refactor: update gcsimage 

### PR DESCRIPTION
**Pull request by nickhsine**
### Notable Changes
1. finalize supported image file type: add `image/bmp` type
2. fix alert message not showing
3. ask end users to check slack channel after uploading image. (also see PR [refactor: update models/Image.js. add slack channel notice](https://github.com/twreporter/plate/pull/174))
4. enable search by iptc metadata

### CODE REVIEW

**Comment by nickhsine** [comment]
Result of commit 8757681

**Review by taylrj — APPROVED** [review]
LGTM 👍

**Comment by nickhsine** [comment]
updated.
@taylrj  please see the latest commit. 
This commit is the searching-by-iptc feature.

File: fields/types/gcsimage/GcsImageColumn.js [hunk 1]
```diff
@@ -33,10 +33,34 @@ var GcsImageColumn = React.createClass({
...
+    const iptc = _.get(value, ['iptc'], {})
+    const { byline, caption, created_time, keywords } = iptc
+
+    let createdTime = created_time
+    if (typeof createdTime === 'string' && createdTime !== '') {
+      const hour = createdTime.slice(0, 2);
+      const minute = createdTime.slice(2, 4);
+      const second = createdTime.slice(4, 6);
+
+      if (hour < 12) {
```
Comment (babygoat) [inline fields/types/gcsimage/GcsImageColumn.js:45]: minor. 如果hour是24小時制的話，我們還需要標示上午下午嗎？

Comment (nickhsine) [reply fields/types/gcsimage/GcsImageColumn.js:45]: 可以不用加

**Comment by nickhsine** [comment]
Updated.
If there is no other comments, I would like to merge it and test staging integration.

**Comment by babygoat** [comment]
LGTM

**Pull request merged**